# Explorar datos iniciales

In [1]:
import pandas as pd

## Poblacion con servicios de salud

In [2]:
df = pd.read_csv("/Users/lucasgarcia/Desktop/Dataton/demanda_servicios_salud_cdmx/data/raw/poblacion_con_servicios_de_salud.csv", encoding="latin-1")

In [3]:
print(df.head())

   _id               alcaldia  pob_sin_ss  pob_con_ss  pob_con_imss  \
0       1           AZCAPOTZALCO   20.909059   78.967620     55.262433   
1       2               COYOACAN   26.196075   73.695372     44.440936   
2       3  CUAJIMALPA DE MORELOS   24.256498   75.439854     40.639269   
3       4      GUSTAVO A. MADERO   25.644671   74.276325     47.342526   
4       5              IZTACALCO   27.209627   72.666576     45.333399   

   pob_con_issste  pob_con_issste_est  pob_con_sspdm  pob_con_bienestar  \
0       11.297185            0.208697       2.683912           7.952014   
1       15.729754            0.086256       1.078043           8.216331   
2        7.327986            0.156188       0.704685          12.810195   
3       11.717295            0.174884       0.607661          12.466261   
4       14.184756            0.136646       0.644683          10.696450   

   pob_con_imssbienestar  pob_con_sspriv  pob_con_ssotro  
0               0.202682        2.541155     

_id               alcaldia  pob_sin_ss  pob_con_ss  pob_con_imss  \
0       1           AZCAPOTZALCO   20.909059   78.967620     55.262433   
1       2               COYOACAN   26.196075   73.695372     44.440936   
2       3  CUAJIMALPA DE MORELOS   24.256498   75.439854     40.639269   
3       4      GUSTAVO A. MADERO   25.644671   74.276325     47.342526   
4       5              IZTACALCO   27.209627   72.666576     45.333399   

   pob_con_issste  pob_con_issste_est  pob_con_sspdm  pob_con_bienestar  \
0       11.297185            0.208697       2.683912           7.952014   
1       15.729754            0.086256       1.078043           8.216331   
2        7.327986            0.156188       0.704685          12.810195   
3       11.717295            0.174884       0.607661          12.466261   
4       14.184756            0.136646       0.644683          10.696450   

   pob_con_imssbienestar  pob_con_sspriv  pob_con_ssotro  
0               0.202682        2.541155        0.660335  
1               0.225569        5.981151        0.822528  
2               0.489237       16.073611        0.899461  
3               0.182895        1.730599        1.335747  
4               0.425259        2.115914        0.664698